babel.config.js

```diff
@@ -126,21 +114,11 @@ module.exports = function(api) {
           "codemods/*/test",
           "eslint/*/src",
           "eslint/*/test",
+          "**/node_modules",
+          "packages/babel-preset-env/data",
         ],
         sourceType: "unambiguous",
       },
-      {
```

Comment:
They are removed since `rollup-plugin-babel` can combine the helpers into a single block.